### ISSUE TITLE
0.7b2 Java version issue?

### ISSUE THREAD

**Issue by jakeopolis** [tvrenamer/tvrenamer#86]
Been running version 0.6 for ages with no trouble. Just tried to run version 0.7b2 on my Mac (OS 10.10.4) and application crashes on launch. Here is terminal output:

Exception in thread "Thread-1" java.lang.UnsupportedClassVersionError: com/google/code/tvrenamer/view/UIStarter : Unsupported major.minor version 51.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:621)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:141)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:283)
	at java.net.URLClassLoader.access$000(URLClassLoader.java:58)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:197)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:301)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)

I am running Java version 8 update 45 (build 1.8.0_45-b14).

**Comment by vipuldelwadia** [comment]
That error sounds like you are trying to run tvrenamer with Java 6. Perhaps you have both installed and Java 6 is being used by default?

**Comment by jakeopolis** [comment]
OK, did some research based on what you said - turns out I did have Java 6 installed as well as the newest version, and I suspect most Mac users will also have this problem. The installer for Java/JRE from Oracle installs Java here to /Library/Internet Plug-Ins/... But a 10.10 Yosemite update from Apple installs Java 6 to System/Library/Java. I removed that legacy version, which crippled all my Java apps. Turns out to run most Java applications on Mac, you need the JDK and not JRE. 

Bottom line: manually removed Java version 6 from System/Library/Java, then installed version 8 JDK, and everything working fine :)

**Comment by vipuldelwadia** [comment]
Marking as closed as per above comment.
